packages/components/generateFlowTypes.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 
 for i in $(find dist/* -name "*.d.ts"); do
-    [ -f "$i" ] || break
+    if  [ ! -f "$i" ] || \
+        [[ "$i" == *".stories."* ]] || \
+        [[ "$i" == *".spec."* ]]
```

Comment:
this isn't technically necessary for the `dist/` folder, but the `flowgen` script is a little slow so it's nice to skip these if we can